vision_evaluation/evaluators.py

```diff
@@ -266,6 +268,11 @@ def _get_id(self):
     def _calculate(self, targets, predictions, average):
         return sm.precision_score(targets, predictions, average=average)
 
+    def get_report(self, **kwargs):
+        average = kwargs.get('average', 'macro')
+        filter_out_zero_tgt = kwargs.get('filter_out_zero_tgt', False)
```

Comment:
I thought that you mean filter_out_zero_tgt should never be True for precision evaluator or is should not be true only when average = samples?

Reply:
but anyway, do we need user to specify this or it is decided once you know what evaluator and what 'average' is?

Reply:
In my opinion it should always be False, but I didn't want to remove the option from the user

Reply:
based on the other thread, I will remove the option here and hard code it to False